dist/algoliasearch.helper.js

```diff
@@ -13727,7 +13727,7 @@ var version = require('./version');
  */
 function AlgoliaSearchHelper(client, index, options) {
   if (client.addAlgoliaAgent && !doesClientAgentContainsHelper(client)) {
-    client.addAlgoliaAgent('JS Helper ' + version);
+    client.addAlgoliaAgent('JS Helper (' + version + ')');
```

Comment:
The `dist/` files are generated, they're not supposed to be edited manually. You can revert these modifications.

Reply:
Done